### ISSUE TITLE
got a calculation error when softmax operator with input shape[1, 1023], axis=-2 

### ISSUE THREAD

**Issue by zjusaga** [onnx/onnx#3244]
as describe in the title,  i got all 1 with numpy and tensorflow. but it seems  that onnx treat this situation with axis=1

**Comment by jcwchen** [comment]
Hi @zjusaga,
Are you saying that the axis is different from before/after tensorflow-onnx conversion?

**Comment by jcwchen** [comment]
Close it due to inactivity.... Please reopen it if you still have questions related to ONNX. Thanks!
